main.cpp

```diff
@@ -1,475 +1,581 @@
+    int numSheep = 1234;
+    //number of cows (int)
+    int numCows = 321;
+    //crop yeild (double)
+    double cropYield = 76543.21;
+    //number of farmers (int)
+    int numFarmers = 6;
+    //area in hectare (double)
+    double farmLandAreaInHectare = 84.3;
+
+    //sell sheep
+    float getTurnoverSheepSale(float priceOfSheep, int numSheep); //returns the income from the sale of this farms sheep
+    //harvest crop
+    void harvestCrop(bool isCropMature, int numFarmHands, bool machinesFueled);
+    //construct new barn
+    void constructNewBarn(bool needNewBarn, float newConstructionBudget);
+};
 
 /*
-Thing 6)
+Thing 4) Commerical Aeroplane
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of engines (int)
+    2) top speed (double)
+    3) number of seats (int)
+    4) fuel tank capacity (double)
+    5) flight range (double)
 3 things it can do:
-    1)
-    2)
-    3)
- */
+    1) take off
+    2) emergency landing
+    3) inflight entertainment
+*/
+
+struct CommericalAeroplane
+{
+    //number of engines (int)
+    int numEngines = 4;
+    //top speed (double)
+    double maxAirSpeedMPH = 576.43;
+    //number of seats (int)
+    int numSeats = 524;
+    //fuel tank capacity (double)
+    double fuelTankCapacityInLitres = 238840.04;
+    //flight range (double)
+    double flightRangeInKM = 14310.23;
+
+    struct Cockpit
+    {
+        float age = 1.35f;
+        int numSeats = 5;
+        int numWindows = 4;
+        bool hasBlackBox = true;
+        bool ElectronicFlightInstrumentSystem = true;
+
+        void increaseAirSpeed(float deltaV, float currentAirSpeed);
+        void groundProximityWarningAlarm(bool nearGround, float proximityWarningThreshold);
+        void autopilotDisengagement(bool pilotAndCopilotChecklistComplete);
+    };
+
+
+    //take off
+    bool takeOffComplete(float groundSpeed, bool landingGear, bool armDoorsAndCrossCheck); //returns whether has this instance of Commercial Areoplane completed a successful take off
+    //emergency landing
+    void emergencyLanding(bool enginefailure, float urrentAirSpeed);
+    //inflight entertainment
+    void inflightEntertainment(int passengerSeatNumber, std::string movieSelction);
+};
 
 /*
-Thing 7)
+Thing 5 ) Player
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) body weight (float)
+    2) sponsor (std:string)
+    3) number of games played (int)
+    4) scoring average (float)
+    5) name (std::string)
 3 things it can do:
-    1)
-    2)
-    3)
- */
+    1) run
+    2) score
+    3) train
+*/
+
+struct Player
+{
+    //body weight (float)
+    float bodyMassKG = 101.34f;
+    //sponsor (std:string)
+    std::string sponsor = "Nike";
+    //number of games played (int)
+    int numGamesPlayed = 104;
+    //scoring average (float)
+    float scoringAverage = 1.78f;
+    //name (std::string)
+    std::string name = "Ronaldo";
+
+    //run
+    float getDistanceRun(int howFast, float howFar, bool startWithLeftFoot);
```

Comment:
Your function names do not match the plain-english descriptions.
They should match almost 1:1
- [ ] revise